src/helm/benchmark/run_specs.py

```diff
@@ -2268,6 +2268,25 @@ def get_cleva_spec(task: str, version: str, subtask: str = None, method: str = A
             max_tokens=20,
         )
         metric_specs = get_exact_match_metric_specs() + get_generative_harms_metric_specs()
+    elif task in ["pinyin_transliteration"]:
+        adapter_spec = get_generation_adapter_spec(
+            instructions=prompt_setting.instructions,
+            input_noun=prompt_setting.input_noun,
+            newline_after_input_noun=prompt_setting.newline_after_input_noun,
+            output_noun=prompt_setting.output_noun,
+            newline_after_output_noun=prompt_setting.newline_after_output_noun,
+            max_train_instances=5,  # limited by the context length
+            max_tokens=150,
+        )
+        metric_specs = get_basic_metric_specs(["chinese_bleu_1"]) + get_generative_harms_metric_specs()
+    elif task in ["classical_chinese_understanding"]:
```

Comment:
I think you can simply add "classical_chinese_understanding" to the list of "text_classification"